spec/rubocop/cop/style/if_unless_modifier_spec.rb

```diff
@@ -11,39 +11,70 @@
     RuboCop::Config.new(hash)
   end
 
-  it 'registers an offense for multiline if that fits on one line' do
-    # This if statement fits exactly on one line if written as a
-    # modifier.
-    condition = 'a' * 38
-    body = 'b' * 36
-    expect("  #{body} if #{condition}".length).to eq(80)
+  context 'multiline if that fits on one line' do
+    let(:source) do
+      ["  if #{condition}",
```

Comment:
@bbtatsov Do you mean just cleaning up the spec or is there something else I can't see? :)